_docs/4-kubernetes/3-bookinfo.md

```diff
@@ -0,0 +1,200 @@
+> resiliency testing with Gremlin. These steps may be added in the future.
+>
+> Before attempting to run this sample, please read through the [prerequisites and caveats](/docs/kubernetes-integration-intro.html#prerequisites-caveats)
+
+## Running the Bookinfo Sample Application
+
+1. Bring up the [control plane](/docs/kubernetes-integration-control-plane.html#deploy)
+
+1. Bring up the application containers:
+
+   ```bash
+   $ kubectl create -f examples/k8s-bookinfo.yaml
+   ```
+
+   The above command automatically launches the gateway service and the bookinfo application microservices.
+   ![Bookinfo app](/docs/figures/amalgam8-example-app-bookinfo.svg)
+   Each microservice has a corresponding Kubernetes service associated with it.
+   Except for the *reviews* microservice, all other microservices are backed by a single pod.
+   The *reviews* microservice has three backing pods, each implementing a different version of the service.
+
+   Confirm that all services and pods are correctly defined and running:
+
+   ```bash
+   $ kubectl get services
+   NAME          CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+   details       None            <none>        9080/TCP         1m
+   gateway       10.102.224.69   <nodes>       6379:32000/TCP   1m
+   kubernetes    10.96.0.1       <none>        443/TCP          22d
+   productpage   None            <none>        9080/TCP         1m
+   ratings       None            <none>        9080/TCP         1m
+   reviews       None            <none>        9080/TCP         1m
+   ```
+
+   and
+
+   ```bash
+   $ kubectl get pods
+   NAME                   READY     STATUS    RESTARTS   AGE
+   details-v1-w16wg       1/1       Running   0          1m
+   gateway-j6vr6          1/1       Running   0          1m
+   productpage-v1-chmvt   2/2       Running   0          1m
+   ratings-v1-2p06t       1/1       Running   0          1m
+   reviews-v1-v39cg       1/1       Running   0          1m
+   reviews-v2-3xk9f       2/2       Running   0          1m
+   reviews-v3-kcwpp       2/2       Running   0          1m
+   rules-56tf8            1/1       Running   0          18h
+   ```
+
+
+1. Expose the Gateway service
+
+   Determine the node on which the `gateway` pod runs, and use the node's IP address as the external gateway IP.
+
+   ```bash
+   $ kubectl describe pod gateway-j6vr6 | grep Node
+   Node:		node4.k8s.example.com/9.147.221.77
+   $ export GATEWAY_URL=9.147.221.77:32000
+   ```
+
+### Content Based Routing
+
+1. Set the default routes to v1 of each service
+
+   ```bash
+   $ kubectl create -f examples/k8s-bookinfo-default-routes.yaml
+   routingrule "set-productpage-v1" created
+   routingrule "set-details-v1" created
+   routingrule "set-ratings-v1" created
+   routingrule "set-reviews-v1" created
+   ```
+
+   You can confirm the routes are defined and valid by inspecting the rules' `Status` field returned in:
```

Comment:
Syntax validity only, nothing about the distribution status. 
Will add the required statement regarding propagation delay of rules.